lib/utils/avatar.vue

```diff
@@ -32,8 +32,14 @@
         }
       },
       imageURL() {
-        // only display the image if it's not the default google avatar
-        if (this.url && this.url.length && this.url !== 'https://lh3.googleusercontent.com/-XdUIqdMkCWA/AAAAAAAAAAI/AAAAAAAAAAA/4252rscbv5M/photo.jpg?sz=50') {
+        // only display the image if it's not the default google avatar (or the built-in clay avatar)
+        if (this.url && this.url === 'kiln-clay-avatar') {
+          // special kiln avatar
+          // note: we're referencing it from amphora-search instead of hardcoding it so we can
```

Comment:
isn't this... hardcoding it???

Reply:
"in amphora-search, we're referencing <giant blob of base64-encoded image> rather than hardcoding it"